plugins/newsdownloader.koplugin/main.lua

```diff
@@ -1,4 +1,4 @@
-local DataStorage = require("datastorage")
+﻿local DataStorage = require("datastorage")
```

Comment:
Did you perhaps accidentally change the line endings? (Sorry, I'm just on the GH website atm, so I can't check.) Looks good besides that. :-)

Reply:
Hmm, that's strange. I have checked the raw file file and realized that it was saved as utf8 with bom. I have corrected it.